src/components/pages/Home/RenderHomePage.js

```diff
@@ -155,7 +154,6 @@ function RenderHomePage(props) {
   };
 
   const deleteBookmark = caseID => {
-    // only works for cases, judge requires name instead of ID to delete
```

Comment:
Good eye removing some of these old comments - cleaning up as we go will make it easier to leave a clean codebase for the next team of developers to work on it!